packages/client/src/runtime/RequestHandler.ts

```diff
@@ -40,13 +41,13 @@ export type RequestParams = {
   callsite?: CallSite
   rejectOnNotFound?: RejectOnNotFound
   transaction?: PrismaPromiseTransaction
-  customDataProxyHeaders?: Record<string, string>
   extensions: MergedExtensionsList
   args?: any
   headers?: Record<string, string>
   unpacker?: Unpacker
   otelParentCtx?: Context
   otelChildCtx?: Context
+  customFetch?: (fetch: Fetch) => Fetch
```

Comment:
Could we maybe name it `customDataProxyFetch` or something similar, to indicate that it won't be used for binary engine?